velox/docs/functions/spark/bitwise.rst

```diff
@@ -10,7 +10,19 @@ Bitwise Functions
 .. spark:function:: bitwise_or(x, y) -> [same as input]
 
     Returns the bitwise OR of ``x`` and ``y`` in 2's complement representation.
+    Corresponds to Spark's operator ``|``.
```

Comment:
nit: could we also document the supported types of bitwise_and and bitwise_or?

Reply:
updated, thanks.